recipes/aplus/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "aplus" %}
+{% set version = "0.11.0" %}
+{% set sha256 = "4f57025413bd9611fb54782b8f46946ea874ae7bb6dd876c0ec45c6a8d60f6e7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install  --record record.txt
```

Comment:
doesn't seem to run that way, doesn't recongnize the --single-version-externally-managed, i think because it imports distutils, is that a problem?